apps/src/applab/applab.js

```diff
@@ -1722,9 +1722,9 @@ Applab.getIdDropdown = function (filterSelector) {
  * @private
  */
 Applab.getIdDropdownFromDom_ = function (documentRoot, filterSelector) {
-  var designModeVizChildren = documentRoot.find('#designModeViz').children();
-  var elements = designModeVizChildren.toArray().concat(
-      designModeVizChildren.children().toArray());
```

Comment:
the problem with pulling these IDs from design mode was that they were wrapped in the draggable/resizable wrapper. It seems easier to just pull these from divApplab which will have been recently updated from designModeViz. As a bonus, students who click the dropdown after the program has run will additionally see dynamically created element ids.